Add group prefix example and table entry (#1822)
sensu/sensu-docs: content/sensu-go/5.13/reference/rbac.md

required     | true 
type         | String
example      | {{< highlight shell >}}"name": "alice"{{< /highlight >}}

> example with prefix | {{< highlight shell >}}"name": "ladp:alice"{{< /highlight >}}

Reviewer: ```suggestion
example with prefix | {{< highlight shell >}}"name": "ad:alice"{{< /highlight >}}
```

Reviewer: Just suggest changing to `ad` to be consistent with the group prefix examples you give later.

}
{{< /highlight >}}

### Role and role binding examples with a group prefix

> If a [groups prefix][38] of `ad` has been configured for [Active Directory authentication][39], the following role and role binding give a `dev` group access to create and manage Sensu workflows within the `default` namespace.

Reviewer: ```suggestion
In the following code example, if a [groups prefix][38] of `ad` is configured for [Active Directory authentication][39], this role and role binding will give a `dev` group access to create and manage Sensu workflows within the `default` namespace.
```

[35]: #cluster-role-bindings
[36]: ../../sensuctl/reference#creating-resources-across-namespaces
[37]: ../license

> [38]: ../../installation/auth/#groups-prefix

Reviewer: This links to the `groups_prefix` in the LDAP spec (https://docs.sensu.io/sensu-go/5.14/installation/auth/#spec-attributes)

Since this example uses AD, I think we would need to add something like `<a name="ad-groups-prefix"></a>` just before the AD `groups-prefix` description in https://docs.sensu.io/sensu-go/5.14/installation/auth/#active-directory-spec-attributes, right?

I guess we could also revise the role/role binding with groups prefix examples section to use LDAP rather than AD, but I assume there is a good reason you used AD there. What do you think?

Author: There seems to already be a `<a name="ad-groups-prefix"></a>` before the groups prefix. So https://docs.sensu.io/sensu-go/5.14/installation/auth/#ad-groups-prefix works already!

I'll make the change so we use AD everywhere.
